api/host.go

```diff
@@ -32,7 +32,7 @@ type (
 	// to /host/storage - a bunch of information about the status of storage
 	// management on the host.
 	StorageGET struct {
-		Folders []modules.StorageFolderMetadata `json:"folders"`
+		FolderMetadata []modules.StorageFolderMetadata `json:"foldermetadata"`
```

Comment:
what would it mean to return the "actual folders?"
I think "folders" is clear enough.